api/api.proto

```diff
@@ -579,3 +579,26 @@ message ActiveRequests {
   int64 RequestCount = 1;
   repeated ActiveRequest ActiveRequest = 2;
 }
+
+// GroupSnapCreateRequest specifies a request to create a snapshot of given group.
+// swagger:parameters snapVolumeGroup
+message GroupSnapCreateRequest {
+  // group id
+  string id = 1;
```

Comment:
nit: id is lower case here and below

Reply:
when doing make Proto it becomes upper case

Reply:
Also, Could you change it to <group><call> style like CloudBackupDelete does it. So this would be `SnapshotGroupCreateXXX` and so on. That would be more consistent with the rest of the APIs